Resources/public/js/autocomplete/50_autocomplete.js

```diff
@@ -142,7 +142,9 @@ jQuery(function($, undefined) {
 					},
 					select: function(event, ui) {
 						this.value = ui.item.label;
-													$(this).trigger('change');
+						$(this).trigger('change');
+						me.val(ui.item.value);
+						me.trigger('change');
```

Comment:
Could you write it in 1 line please?
`me.val(ui.item.value).trigger('change');`